src/ops.jl

```diff
@@ -335,3 +335,9 @@ Base.Test.approx_full(a::FixedArray) = a
 +{m, n, T}(J::Base.LinAlg.UniformScaling, A::Mat{m,n, T}) = A + J
 -{m, n, T}(A::Mat{m,n, T}, J::Base.LinAlg.UniformScaling) = A + (-J)
 -{m, n, T}(J::Base.LinAlg.UniformScaling, A::Mat{m,n, T}) = J.λ*eye(Mat{m,n,T}) - A
+
+# typed floor, ceil, round, trunc
+Base.floor{T}(::Type{T}, A::FixedArray) = map(x->floor(T, x), A)
+Base.ceil{T}( ::Type{T}, A::FixedArray) = map(x->ceil( T, x), A)
+Base.trunc{T}(::Type{T}, A::FixedArray) = map(x->trunc(T, x), A)
+Base.round{T}(::Type{T}, A::FixedArray) = map(x->round(T, x), A)
```

Comment:
I guess this will be pretty slow in 0.4 if we use anonymous functions.  I know an 0.5 RC is coming soon but perhaps we shouldn't give up on 0.4 just yet?